src/test/test-blockdev-util.c

```diff
@@ -26,6 +38,33 @@ static void test_path_is_encrypted_one(const char *p, int expect) {
         assert_se(expect < 0 || ((r > 0) == (expect > 0)));
 }
 
+TEST(get_block_device) {
+        _cleanup_(sd_device_unrefp) sd_device *sd_dev = NULL;
+        _cleanup_free_ char *dev_path = NULL;
+        int r;
+        dev_t devnum;
+        struct stat st;
+
+        dev_path = get_device_path();
+
+        /*r = get_block_device(dev_path, &devnum);
+        ASSERT_OK(r);*/
+
+        ASSERT_OK(stat(dev_path, &st) == 0);
+        ASSERT_OK(S_ISBLK(st.st_mode));
+        devnum = (unsigned long) st.st_rdev;
+
+        r = sd_device_new_from_devnum(&sd_dev, 'b', devnum);
+        ASSERT_OK(r);
+
+        r = device_is_devtype(sd_dev, "disk");
+        ASSERT_OK(r);
+
+        sd_device *parent = NULL;
+        r = sd_device_get_parent(sd_dev, &parent);
+        ASSERT_OK(r);
```

Comment:
All these are in sd-device, and we already have test cases for them in test-sd-device. 

This adds nothing about blockdev-util.